app/utils/tabManager.js

```diff
@@ -24,11 +25,12 @@ const OPEN_TAB_ID_KEY = 'openTabId';
  *
  * Note: this may cause two copies of Yoroi loading at the same time close each other
  */
-export function addCloseListener(window) {
+export function addCloseListener(window: any) {
   window.onstorage = (e: StorageEvent) => {
     // if another Yoroi tab open, close this tab
     if (e.key === OPEN_TAB_ID_KEY) {
       // note: we don't need "tabs" permission to get or remove our own tab
+      // $FlowFixMe
```

Comment:
I think this can be [avoided by declaring](https://github.com/facebook/flow/issues/1424) chrome: `declare var chrome;`

Reply:
doing `declare var chrome` just silences the error same way `$FlowFixMe` does except it does it for the whole file instead of a single line. It doesn't actually add the missing  type definitions.